packages/react-composites/src/composites/CallComposite/adapter/AzureCommunicationCallAdapter.ts

```diff
@@ -775,7 +778,7 @@ export const useAzureCommunicationCallAdapter = (
       })();
     },
     // Explicitly list all arguments so that caller doesn't have to memoize the `args` object.
-    [adapterRef, afterCreateRef, beforeDisposeRef, credential, displayName, locator, userId]
+    [adapterRef, afterCreateRef, alternativeCallerId, beforeDisposeRef, credential, displayName, locator, userId]
```

Comment:
Did you forget to add `/* @conditional-compile-remove(PSTN-calls) */` in dependency array to remove `alternativeCallerId`?

Reply:
yup! oops!